swap_meet/vendor.py

```diff
@@ -1,2 +1,74 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category_name):
+        res = []
+        for item in self.inventory:
+            if item.category == category_name:
+                res.append(item)
+        return res
```

Comment:
What does `res` isn't a very descriptive name. We can use names like `category_items` or even `items` for the list name to better label what this list is supposed to hold. 

Additionally, we can use list comprehension to make this logic shorter.
```python
res = [item for item in self.inventory if item.category == category_name]
return res
```